sonar-project.properties

```diff
@@ -1,18 +1,21 @@
-# must be unique in a given SonarQube instance
+#----- Default SonarQube server
 sonar.host.url=http://localhost:9000
 
-sonar.projectKey=org.sonarqube:cvs-svc-authoriser
+# must be unique in a given SonarQube instance
+sonar.projectKey=org.sonarqube:cvs-svc-activities
 
 # this is the name and version displayed in the SonarQube UI. Was mandatory prior to SonarQube 6.1.
-sonar.projectName=cvs-svc-authoriser
```

Comment:
Don't know why it was changed, but I believe having the wrong name will make it fail in the pipeline

Reply:
Thanks @gjulien-bjss  🙂